taichi/python/export_lang.cpp

```diff
@@ -752,6 +752,12 @@ void export_lang(py::module &m) {
              return expr->cast<FieldExpression>()->snode_grad_type ==
                     SNodeGradType::kPrimal;
            })
+      .def("is_lvalue",
+           [](Expr *expr) {
+             return expr->is<IdExpression>() || expr->is<IndexExpression>() ||
+                    (expr->is<ArgLoadExpression>() &&
+                     expr->cast<ArgLoadExpression>()->is_lvalue());
```

Comment:
Why not directly return `expr->is_lvalue()`?

Reply:
Thanks! I’ll fix it.